GUI/Types/Renderer/WorldLoader.cs

```diff
@@ -908,47 +817,18 @@ void LoadEntity(string classname, Entity entity)
                 var isAnimated = modelNode.SetAnimationForWorldPreview(animation);
                 if (isAnimated)
                 {
-                    var holdAnimation = entity.GetProperty("holdanimation");
-                    if (holdAnimation != default)
+                    var holdAnimationOn = entity.GetPropertyUnchecked<bool>("holdanimation");
```

Comment:
does this actually work for strings of "1"

Reply:
Yes, `GetPropertyUnchecked` now converts the string to float, which `Convert.ChangeType` is able to convert to boolean.

![image](https://github.com/user-attachments/assets/b224028b-6f5f-41bd-8bfd-5c0f2d03c83e)